upgrades/steps_20.go

```diff
@@ -36,6 +36,16 @@ func stepsFor20() []Step {
 			targets:     []Target{Controller},
 			run:         removeCharmGetCache,
 		},
+		&upgradeStep{
```

Comment:
These upgrade steps are in the wrong place. There should be some for 2.2. These are for 2.0.